tests/subyt/test_conditional.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import time
+import unittest
+from pathlib import Path
+
+import pandas as pd
+from conftest import run_single_test
+
+from sema.commons.log.loader import load_log_config
+from sema.subyt import Subyt
+
+load_log_config()
```

Comment:
tests should only `load_log_config` through run_single_test call in the `main()`

seems this needs to be applied to all tests